src/App.js

```diff
@@ -1,122 +1,99 @@
-                                    <span className="bold">Type:</span>{" "}
-                                    {pokemonInfo.type}
-                                </p>
-                                <p>
-                                    <span className="bold">HP:</span>{" "}
-                                    {pokemonInfo.hp}
-                                </p>
-                                <p>
-                                    <span className="bold">Attack:</span>{" "}
-                                    {pokemonInfo.attack}
-                                </p>
-                                <p>
-                                    <span className="bold">Defence:</span>{" "}
-                                    {pokemonInfo.defense}
-                                </p>
-                            </div>
-                            <div className="image-window">
-                                <img
-                                    className="pokemon-image"
-                                    src={pokemonInfo.imageUrl}
-                                    alt="front view of chosen pokemon"
-                                ></img>
-                            </div>
-                        </div>
-                    </div>
-                )}
-            </div>
-        </div>
-    );
+  }, []);
+
+  const handleChosen = () => {
+    if (pokemonName) {
+      setIsChosen(true)
+    }
+  }
+
+  useEffect(() => {
+    if (location.pathname !== "/") {
+      setIsChosen(true)
+    } else {
+      setIsChosen(false);
+    }
+  }, [location.pathname]);
+
+  return (
+    <div className="App">
+      <div className="title-section">
+        <h1 className="title">Poke Stats</h1>
+        <input
+          className="search-input"
+          type="text"
+          onChange={(event) => {
+            setPokemonName(event.target.value);
+          }}
+        ></input>
+        <Link to={{ pathname: `/${pokemonName}`}}>
+          <button id="search-button" className="search-button" onClick={handleChosen}>
+            Search Pokemon
+          </button>
+        </Link>
+      </div>
+      {isChosen ? (
+        <Outlet />
+      ) : (
+        <h1 className="call-to-action">Please Choose a Pokemon</h1>
+      )}
+    </div>
+  );
 }
 
 export default function App() {
-    return (
-        <div className="wrapper">
-            <BrowserRouter>
-                <Routes>
-                    <Route path="/" exact element={<PokeApp />} />
-                    <Route path="/pokeapp" exact element={<PokeApp />} />
-                    <Route path="/404" exact element={<FourOhFour />} />
-                </Routes>
-            </BrowserRouter>
-        </div>
-    );
+  return (
+    <div className="wrapper">
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" exact element={<PokeApp />}>
+            <Route path="/:pokemonName" exact element={<PokemonDetailsPage />} />
+          </Route>
```

Comment:
Following on from above about index routes, I would change this to have something like:

```suggestion
          <Route path="/" exact element={<PokeApp />}>
            <Route index element={<PokemonIndexPage />} />
            <Route path="/:pokemonName" exact element={<PokemonDetailsPage />} />
          </Route>
```

Where `<PokemonIndexPage>` is reponsible for showing all the content that should be visible when you are on the path `/` (the index path)

Reply:
Likewise we talked through this and it's now set up and works.